Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -115,17 +101,52 @@ private extension TodosViewController {
         return [uncompleted, completed]
     }
     
-    func reloadData() {
-        let newlyCompleted = todos[TodosIndex.uncompleted.rawValue].filter { $0.completed }
-        if !newlyCompleted.isEmpty {
-            todos[TodosIndex.uncompleted.rawValue] = todos[TodosIndex.uncompleted.rawValue].filter { $0.completed == false }
-            todos[TodosIndex.completed.rawValue] += newlyCompleted
+    func didSelectCheckBox(atIndexPath indexPath: IndexPath) -> (() -> Void)? {
```

Comment:
This is unnecessary - lets just have a method that contains the code that you return here as a `closure`.